src/main/java/org/saltyrtc/client/signaling/InitiatorSignaling.java

```diff
@@ -503,12 +503,12 @@ private void sendAuth(Responder responder, SignalingChannelNonce nonce) throws S
     /**
      * Drop specific responder.
      */
-    private void dropResponder(Responder responder, @Nullable Integer reason) throws SignalingException, ConnectionException {
-        final DropResponder msg = new DropResponder(responder.getId(), reason);
-        final byte[] packet = this.buildPacket(msg, responder);
-        this.getLogger().debug("Sending drop-responder " + responder.getId());
+    private void dropResponder(final short responderId, @Nullable Integer reason) throws SignalingException, ConnectionException {
```

Comment:
Any reason to change the signature from `Responder` to `short`? While it works just as well, it reduces type safety because you could now accidentally swap responder id and reason code when calling the function.

Reply:
There's not always a `Responder` instance when dropping a responder. See `handleNewResponder`.

Reply:
Ok, makes sense :slightly_smiling_face: I overlooked that.